features/ticketing/daemon/src/test/java/org/opennms/netmgt/ticketd/DefaultTicketerServiceLayerIT.java

```diff
@@ -200,6 +201,7 @@ public void testTicketsForSituations() throws PluginException {
         assertFalse(relatedAlarms.isEmpty());
         assertEquals(relatedAlarms.get(0).getNodeId(), testNode.getId());
 
+        m_alarmDao.delete(situation); // XXX this is working around a bug with deleting an alarm that is part of a situation
```

Comment:
Do you have more info on this bug?  Is there an open issue about it?

Reply:
turns out this is a cascade issue with the alarm association object, I _think_ I have a fix